pkgs/development/python-modules/ssdpy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+,
+}:
+buildPythonPackage rec {
+  pname = "ssdpy";
+  version = "0.4.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-SXHGoBD3fxR+zb7JWT8tkYfD+2Nliw9ewI9Me+I4dCU=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  meta = with lib; {
+    changelog = "https://github.com/MoshiBin/ssdpy/releases/tag/${version}";
+    description = "A lightweight, compatible SSDP library for Python.";
+    homepage = "https://github.com/MoshiBin/ssdpy";
+    license = licenses.mit;
+    maintainers = [ ];
```

Comment:
You don't want to maintain this?

Reply:
I will maintain it if that's what's required, but I don't really know anything about the library. I just needed it as a dependency of controku that wasn't already packaged.

Reply:
Usually there is no need to be an expert to maintain a package. 
Further, if this lib is a dependency of your main target (controku), it comes to reason you are interested in keep it up-to-date with that target.

Reply:
Fair point, I'll add myself to it then.